CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## Unreleased
 
+### API breaking Changes
+
+#### Changed APIs
+
+- Rename iOS and MacCatalyst platform specific options from `Cocoa` to `Native` ([#2940](https://github.com/getsentry/sentry-dotnet/pull/2940))
```

Comment:
Note to final 4.0.0 changelog rewording: outside the beta there was no `Cocoa`, it was `iOS` to `Native` (right?)

Reply:
Yes and that is also true for many other parts of the 4.0 changelog. we need to update it and review thoroughly before the final release.

Reply:
We could keep track of these things in a ticket (we already have https://github.com/getsentry/sentry-dotnet/issues/2673)?